cmd/strelaysrv/main.go

```diff
@@ -233,6 +233,7 @@ func main() {
 	uri, err := url.Parse(fmt.Sprintf("relay://%s/?id=%s&pingInterval=%s&networkTimeout=%s&sessionLimitBps=%d&globalLimitBps=%d&statusAddr=%s&providedBy=%s", mapping.Address(), id, pingInterval, networkTimeout, sessionLimitBps, globalLimitBps, statusAddr, providedBy))
 	if err != nil {
 		log.Fatalln("Failed to construct URI", err)
+		return
```

Comment:
Not necessary but 🤷

Reply:
Your fancy lint started crying.

Reply:
Yeah what it means is they the defer doesn’t run due to Fatalln, but they doesn’t matter much

Reply:
🤷‍♂️ 

```
In these cases, in case of failure, defer never gets called.

Example:

defer os.Remove(filename)
if bad {
    log.Fatalf("something bad happened")
}
should be written as:

defer os.Remove(filename)
if bad {
    log.Printf("something bad happened")
    return
}
```

Reply:
Println + return is a normal return so the defer runs. Fatalln is an immediate exit so no defer gets a chance to run.  But the defer is a context cancel so we have no reason to care.